app/services/storage.js

```diff
@@ -18,28 +18,49 @@
  * @module Storage
  */
 
-import * as sqlite from './sqlite';
+import cbmkdirp from 'less-mkdirp';
```

Comment:
Did we already take this dep?  (I don't see it locally, and it's not in this patch.)